firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -143,18 +143,19 @@ public Thread newThread(Runnable r) {
    * or empty.
    */
   private void preConditionChecks() {
-    Preconditions.checkNotEmpty(
-        getApplicationId(),
-        "Please set your Application ID. A valid Firebase App ID is required to communicate "
-            + "with Firebase server APIs: It identifies your application with Firebase.");
-    Preconditions.checkNotEmpty(
-        getProjectIdentifier(),
-        "Please set your project ID. A valid Firebase project ID is required to communicate "
-            + "with Firebase server APIs: It identifies your project with Google.");
-    Preconditions.checkNotEmpty(
-        getApiKey(),
-        "Please set a valid API key. A Firebase API key is required to communicate with "
-            + "Firebase server APIs: It authenticates your project with Google.");
+    Preconditions.checkNotEmpty(getApplicationId());
+    Preconditions.checkNotEmpty(getProjectIdentifier());
+    Preconditions.checkNotEmpty(getApiKey());
```

Comment:
Why did you remove the messages?
This is important for developers to see!
Please re-add!